dubbo-common/src/main/java/org/apache/dubbo/common/utils/DubboAppender.java

```diff
@@ -16,13 +16,18 @@
  */
 package org.apache.dubbo.common.utils;
 
-import org.apache.log4j.ConsoleAppender;
+import org.apache.log4j.FileAppender;
 import org.apache.log4j.spi.LoggingEvent;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public class DubboAppender extends ConsoleAppender {
+public class DubboAppender extends FileAppender {
+
+    public DubboAppender() {
+        super();
+        setFile("dubbo.log");

```

Comment:
should we not read this from constant or from some config?

Reply:
This will initialize output file with a default value.
Usually, we will overwrite it in `log4j.xml`, in this PR we use `<param name="File" value="../../dubbo.log"/>`

Reply:
If that is the case, it is better to add a 

```java
private static final String DEFAULT_FILE_NAME = "dubbo.log";
```